cmake/package.cmake

```diff
@@ -18,6 +18,14 @@ if(USE_DRM)
   pkg_check_modules(LIBSYSTEMD REQUIRED libsystemd)
   set(THREADS_PREFER_PTHREAD_FLAG ON)
   find_package(Threads REQUIRED)
+elseif(USE_EGLSTREAM)
```

Comment:
Why do you use `elseif` instead of `if(USE_DRM OR USE_EGLSTREAM)`?

Reply:
I fixed it.